doc/rst/usingchapel/chplenv.rst

```diff
@@ -385,6 +385,13 @@ CHPL_MEM
    If unset, ``CHPL_MEM`` defaults to ``jemalloc`` for most configurations.
    If the target platform is ``cygwin*`` it defaults to ``cstdlib``
 
+   .. note::
+     Certain ``CHPL_COMM`` settings (e.g. ugni and gasnet segment fast/large)
+     register the heap to improve communication performance.  Registering the
+     heap requires special allocator support that not all allocators provide.
+     Currently ``jemalloc`` is required for configurations that require a
```

Comment:
Department of redundancy department (2 "required"s).  Maybe: "Currently only ``jemalloc`` is capable of supporting configurations that require a registered heap."